conda_forge_tick/executors.py

```diff
@@ -21,16 +21,30 @@ def __exit__(self, *args, **kwargs):
 DRLOCK = DummyLock()
 
 
+@contextlib.contextmanager
+def lock_git_operation():
```

Comment:
But isn't this problematic? Let's say we also want to restrict parallelism for "eating bananas" in the future. Then using a single lock for everything would prevent us to "eat bananas" in thread/process/dask worker A and to run git operations in thread/process/dask worker B at the same time although we most likely want that, which defeats multithreading/-processing.

It would be better to use more locks for different things, considering that one global lock for all git-related operations is a big compromise already.

Reply:
Sure so rename the TRLOCK, PRLOCK and DRLOCK variables then.

Reply:
Done.